eth/downloader/downloader.go

```diff
@@ -1767,6 +1812,10 @@ func (d *Downloader) DeliverSnapPacket(peer *snap.Peer, packet snap.Packet) erro
 	case *snap.TrieNodesPacket:
 		return d.SnapSyncer.OnTrieNodes(peer, packet.ID, packet.Nodes)
 
+	case *snap.HvmLightStatePacket:
+		log.Info("Received hVM Light State packet", "peer", peer.RemoteAddr().String())
```

Comment:
This is another one I worry a little about how noisy for Info.

Reply:
Yeah this one could be noisy, demoting to Debug